README.md

```diff
@@ -25,8 +25,10 @@ Writing your own adapters for currently unsupported analytics services is easy t
       'visitorType': 'high-value'
     }];
     ```
-4. `KISSMetrics` (WIP)
-5. `CrazyEgg` (WIP)
+1. `Segment`
+  - `key`: [Segment key](https://segment.com/docs/libraries/analytics.js/quickstart/)
+1. `KISSMetrics` (WIP)
+1. `CrazyEgg` (WIP)
```

Comment:
Are you thinking of not directly supporting these other services, pushing people to go through Segment to get this functionality? Less to maintain in the repo.